llvm/utils/TableGen/Common/Utils.h

```diff
@@ -0,0 +1,25 @@
+//===- Utils.h - Common Utilities -------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_UTILS_TABLEGEN_UTILS_H
```

Comment:
```suggestion
#ifndef LLVM_UTILS_TABLEGEN_COMMON_UTILS_H
```

Reply:
The other headers have been moved to this subdirectory recently. The author probably just forgot to update include guards.

Reply:
Done.